packs/bestiary-family-ability-glossary/skeleton-bone-powder.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "defensive",
         "description": {
-            "value": "<p>When the skeleton takes physical damage from a critical hit, one of its bones is pulverized into a fine powder.</p>\n<p>All creatures in a @Template[type:emanation|distance:5] that breathe take @Damage[1d6[persistent,poison]] damage (plus an additional 1d6 for every 6 levels the skeleton has).</p>"
```

Comment:
Added this as a leveled link at the end, since putting it inline would have resulted in a very long button.

Reply:
Not sure what you mean. Also this should use `@Damage` so elite / weak work on it properly.

Reply:
I meant that I couldn't figure out a good way to incorporate the link in the main body.

This made it seem like it wasn't auto-increased:
<img width="716" alt="Screen Shot 2023-10-18 at 9 33 00 AM" src="https://github.com/foundryvtt/pf2e/assets/1904698/5c22625d-e006-4181-875c-d5b46cf6a8e8">

And this was too long:
<img width="640" alt="Screen Shot 2023-10-18 at 9 31 52 AM" src="https://github.com/foundryvtt/pf2e/assets/1904698/e13f92aa-1850-4a1d-9f95-a041feef51de">

As far as @Damage, is that appropriate when the ability increases with level? Here's an example with a level 5 skeleton, and then increasing it to 6 via elite:
<img width="298" alt="Screen Shot 2023-10-18 at 9 30 03 AM" src="https://github.com/foundryvtt/pf2e/assets/1904698/86844424-bc13-4e3d-974c-40baee8950d2">

Reply:
When we add these to skeletons in the system we override the damage to be what the stat block says and remove the extraneous text, so it's fine to just use the auto label on it and let it be modified by the GM / us.